### PR TITLE
Create npositions function

### PR DESCRIPTION
**Pull request by Tortar**
Hi! Amazing library! I created a `npositions` function to address the "todo" in `random_empty`. I'm not so used to Julia style yet (coming from a Python background), hope not to have done any major design mistake :-).

Also don't know if I'm wrong but `positions` function for a `GraphSpace` instance didn't work:

```julia
using Agents, Graphs
g = path_graph(6)
space_graph = GraphSpace(g)
println(positions(space_graph)) # this failed
```

so I updated it, but maybe if this is true, it should be updated in another pr.

### CODE REVIEW

**Comment by Tortar** [comment]
Tested the changes with `julia runtests.jl` and locally everything passed

File: src/spaces/graph.jl [hunk 1]
```diff
@@ -84,7 +84,8 @@ function add_agent_to_space!(
 end
 
 # The following is for the discrete space API:
-positions(space::GraphSpace) = 1:nv(space)
+positions(space::GraphSpace) = 1:nv(space.graph)
+npositions(space::GraphSpace) = nv(space.graph)
```
Comment (Datseris) [inline src/spaces/graph.jl:88]:
```suggestion
npositions(space::GraphSpace) = nv(space.graph)
positions(space::GraphSpace) = 1:npositions(space)
```

**Review by Datseris — APPROVED** [review]
Great :) Please add a changelog entry under the next minor release in CHANGELOG.md!

**Comment by Tortar** [comment]
Added to CHANGELOG.md +1:

**Pull request merged**